state/aws.go

```diff
@@ -105,8 +105,10 @@ func (a *AWS) GetStates() (states []string, err error) {
 
 	var keys []string
 	for _, obj := range result.Contents {
-		if strings.HasSuffix(*obj.Key, a.fileExtension) {
-			keys = append(keys, *obj.Key)
+		for _, ext := range strings.Split(a.fileExtension, ",") {
```

Comment:
As stated above, you should be able to retrieve the extensions as a list directly from the config.